lib/rules/unused.ts

```diff
@@ -21,18 +21,19 @@ export class RUnused extends RuleBase implements IRule {
     if (obj.parent instanceof OEntity && obj.rootFile.architectures.find(a => a.entityName.getLText() === (obj.parent as OEntity).lexerToken.getLText()) === undefined) {
       return;
     }
-    if (this.unusedSignalRegex.exec(obj.lexerToken.text) === null) {
+    const token = implementsIHasLexerToken(obj) ? obj.lexerToken : obj.referenceToken;
+    if (this.unusedSignalRegex.exec(token.text) === null) {
```

Comment:
How can a reference token be unused? Whats happening here?

Reply:
This is only for unused components as they no longer have a lexerToken but instead a referenceToken and can be unused...